ext/bg/js/translator.js

```diff
@@ -562,6 +564,18 @@ class Translator {
         return tagMetaList;
     }
 
+    getFrequencyData(expression, data, dictionary, term) {
+        if (data !== null && typeof data === 'object') {
```

Comment:
Could use isObject as well

Reply:
This one I think is fine to leave as-is; the `Array.isArray` check is not necessary for this case since it should have been validated during import. Additionally, even if somehow the value was an array, it won't be any more wrong than having an incorrectly formatted object.